plugins/inputs/x509_cert/x509_cert_test.go

```diff
@@ -199,30 +197,30 @@ func TestTags(t *testing.T) {
 	acc := testutil.Accumulator{}
 	require.NoError(t, sc.Gather(&acc))
 
-	assert.True(t, acc.HasMeasurement("x509_cert"))
+	require.True(t, acc.HasMeasurement("x509_cert"))
 
-	assert.True(t, acc.HasTag("x509_cert", "common_name"))
-	assert.Equal(t, "server.localdomain", acc.TagValue("x509_cert", "common_name"))
+	require.True(t, acc.HasTag("x509_cert", "common_name"))
+	require.Equal(t, "server.localdomain", acc.TagValue("x509_cert", "common_name"))
 
-	assert.True(t, acc.HasTag("x509_cert", "signature_algorithm"))
-	assert.Equal(t, "SHA256-RSA", acc.TagValue("x509_cert", "signature_algorithm"))
+	require.True(t, acc.HasTag("x509_cert", "signature_algorithm"))
+	require.Equal(t, "SHA256-RSA", acc.TagValue("x509_cert", "signature_algorithm"))
 
-	assert.True(t, acc.HasTag("x509_cert", "public_key_algorithm"))
-	assert.Equal(t, "RSA", acc.TagValue("x509_cert", "public_key_algorithm"))
+	require.True(t, acc.HasTag("x509_cert", "public_key_algorithm"))
+	require.Equal(t, "RSA", acc.TagValue("x509_cert", "public_key_algorithm"))
 
-	assert.True(t, acc.HasTag("x509_cert", "issuer_common_name"))
-	assert.Equal(t, "Telegraf Test CA", acc.TagValue("x509_cert", "issuer_common_name"))
+	require.True(t, acc.HasTag("x509_cert", "issuer_common_name"))
+	require.Equal(t, "Telegraf Test CA", acc.TagValue("x509_cert", "issuer_common_name"))
 
-	assert.True(t, acc.HasTag("x509_cert", "san"))
-	assert.Equal(t, "localhost,127.0.0.1", acc.TagValue("x509_cert", "san"))
+	require.True(t, acc.HasTag("x509_cert", "san"))
+	require.Equal(t, "localhost,127.0.0.1", acc.TagValue("x509_cert", "san"))
 
-	assert.True(t, acc.HasTag("x509_cert", "serial_number"))
+	require.True(t, acc.HasTag("x509_cert", "serial_number"))
 	serialNumber := new(big.Int)
 	_, validSerialNumber := serialNumber.SetString(acc.TagValue("x509_cert", "serial_number"), 16)
 	if !validSerialNumber {
```

Comment:
How about `require.Truef()`?

Reply:
I don't if I want to go that deep with changing all `True`'s to `Truef`'s...

Reply:
I think my comment was misleading. I meant the
```golang
	_, validSerialNumber := serialNumber.SetString(acc.TagValue("x509_cert", "serial_number"), 16)
	if !validSerialNumber {
		t.Errorf("Expected a valid Hex serial number but got %s", acc.TagValue("x509_cert", "serial_number"))
	}
``` 
part... :-)

Reply:
Makes more sense, changed :)